.github/workflows/debian_10_arm.yml

```diff
@@ -98,9 +99,30 @@ jobs:
         -DENABLE_TESTS=ON \
         -DENABLE_PYTHON=OFF
 
+  CXX_Unit_Tests:
+    name: C++ unit tests
+    needs: [ Docker, Build, Smart_CI ]
+    uses: ./.github/workflows/job_cxx_unit_tests.yml
+    with:
+      runner: 'aks-linux-16-cores-arm'
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_test.debian_10_arm }}
+      affected-components: ${{ needs.smart_ci.outputs.affected_components }}
+      os: 'debian_10'
+
+  CPU_Functional_Tests:
+    name: CPU functional tests
+    if: ${{ 'false' }} # Ticket: 153289
```

Comment:
I tried skipping the failing tests but every subsequent run showed new failing tests: [1](https://github.com/openvinotoolkit/openvino/actions/runs/11141375383), [2](https://github.com/openvinotoolkit/openvino/actions/runs/11142227247), [3](https://github.com/openvinotoolkit/openvino/actions/runs/11142726055) so I think it should be skipped entirely and stabilised separately.